ui/pages/token-allowance/token-allowance.js

```diff
@@ -347,22 +348,18 @@ export default function TokenAllowance({
             name={origin}
             url={siteImage}
           />
-          <Typography
-            variant={TypographyVariant.H6}
-            fontWeight={FONT_WEIGHT.NORMAL}
+          <Text
+            variant={TextVariant.bodySm}
+            as="h6"
             color={TextColor.textAlternative}
-            boxProps={{ marginLeft: 1, marginTop: 2 }}
+            marginLeft={1}
```

Comment:
Should this have a `marginTop` prop to match what the boxProps did?

Reply:
Hey @NidhiKJha , @georgewrmarshall ,
I think there is some confusion in that.

Please check.

Reply:
Ah apologies. That makes sense